### PR TITLE
Add license to feature.xml

### LINKED ISSUE #50: License in feature.xml
Make sure we know what the license is, then add it to the license element in feature.xml

### PR DESCRIPTION
**Pull request by elharo**
@briandealwis 
fix #50

### CODE REVIEW

**Comment by briandealwis** [comment]
The Google Dataflow feature explicitly specifies:

> (c) Copyright Google Inc. 2015. All rights reserved.

I wonder if we should set up a standard shared license feature (com.google.aslv2?)

**Comment by elharo** [comment]
The Dataflow feature is in the legacy plugin, right? 

What would a standard shared license feature involve? This is a bureaucratic fix wihtout a lot of user impact. I don't want to burn too many cycles on it.

**Comment by briandealwis** [comment]
Dataflow is a new plugin.

A shared license contains boilerplate that is copied into any referencing features. It also seems to ensure that the user sees one license when installing.  It actually simplifies things.  Let's punt it into a separate issue and I'm happy to take it. .

**Comment by briandealwis** [comment]
I did author these commits. (Work your magic, googlebot.)

**Comment by briandealwis** [comment]
👍 to the googlebot?

**Comment by briandealwis** [comment]
@googlebot 👍

**Comment by briandealwis** [comment]
PTAL @elharo.  I rebased your commit so I think @googlebot wants to hear from you?

File: features/com.google.licenses.apache_v2/pom.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.tools.eclipse</groupId>
+    <artifactId>trunk</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <relativePath>../..</relativePath>
+  </parent>
+  <artifactId>com.google.licenses.apache_v2</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
```
Comment (elharo) [inline features/com.google.licenses.apache_v2/pom.xml]: 0.1.0

File: features/com.google.cloud.tools.eclipse.suite.e45.feature/feature.xml [hunk 2]
```diff
@@ -3,18 +3,19 @@
...
+      license-feature="com.google.licenses.apache_v2">
 
    <description>
-      Support development of Google Cloud Platform-based Projects in Eclipse 4.5
+      %description
    </description>
 
    <copyright>
-      Copyright 2016
+      %copyright.text
```
Comment (elharo) [inline features/com.google.cloud.tools.eclipse.suite.e45.feature/feature.xml]: I only see description set in .properties. Where are copyright.text, license, and license.text?

File: features/com.google.licenses.apache_v2/feature.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="com.google.licenses.apache_v2"
+      label="Apache License v2"
+      version="0.0.1.qualifier"
```
Comment (elharo) [inline features/com.google.licenses.apache_v2/feature.xml]: 0.1.0

**Comment by briandealwis** [comment]
PTAL @elharo

**Comment by elharo** [comment]
LGTM :+1: 
I'm not sure how to fix the CLA check. I manually tagged it cls: yes.

**Pull request merged**